tests/CMakeLists.txt

```diff
@@ -1,6 +1,4 @@
-# Use our top-level targets as an ordering clue to cmake: the project
-# needs to be built before we can populate anything...
-get_property(TOP_TARGETS DIRECTORY .. PROPERTY BUILDSYSTEM_TARGETS)
+set(RPM_TARGETS librpm librpmio librpmbuild librpmsign)
```

Comment:
After a brief wtf moment, I see that I broke the top level target fetch in commit 86d99a1d20bbebfeb4427c99b1bb1053029c3317. Oof. 

But this is a fine example of what should've gone into a separate PR as soon as noticed because this has nothing to do with the change in *this* PR. I'll fix since it's me who broke it in the first place.

Reply:
Oh, this explains that little mystery of "why on earth doesn't this TOP_TARGETS work for me as expected" :smile: I actually settled on that having been broken from the start and just patched it quickly this way, with the thought of investigating a bit more afterwards. And yes, this would've gone to a separate commit in this PR as well :smile: 

So yup, feel free to fix, I'll then just rebase. Thanks :smile:

Reply:
Submitted as #2692 now. And whenever you run into mysteries like that, just ask :sweat_smile:

Reply:
It's also a good example of bad (variable) naming in the original version: being top-level had nothing to do with it, it was the tools we were after. Had the variable been called "TOOLPROGS" or such from the start, it probably would've been far more obvious as to why it's not working now.